docs/src/prerequisites.md

```diff
@@ -1,5 +1,10 @@
 ## Pre-requisites
 
+1. Local development tools: git, terraform, a terminal.
+2. Understanding of OCI and its services, in particular the OCI Networking services.
+3. Your Terraform user must be part of a group that can
+   - manage VCNs in the compartment you will be creating the VCN and other resources
```

Comment:
This isn't a pre-requisite as you have steps to do this later in the same document.